docs/getting_started.md

```diff
@@ -216,6 +231,76 @@ Here are some resources that can be used to get you started working with `AnnDat
 - [Converting directly from R to Python](https://theislab.github.io/scanpy-in-R/#converting-from-r-to-python)
 - [Homepage for scanpy tutorials](https://scanpy.readthedocs.io/en/latest/tutorials.html)
 
+
+## Special considerations for CITE-seq experiments
+
+### Filtering cells based on ADT quality control
+
+The `SingleCellExperiment` objects stored in both the `filtered.rds` and the `processed.rds` files have not been filtered based on ADT-level statistics, but both objects contain indicator variables to quickly perform this filtering.
+
+In both objects, you can see cells flagged for removal based on ADT-level QC statistics in the alternative experiment's `discard` column as calculated by [`DropletUtils::CleanTagCounts()](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html), where `TRUE` indicates the cell should be removed.
+
+```r
+# See which cells should be removed (`TRUE`) vs. kept (`FALSE`)
+# This column is in both `filtered.rds` and `processed.rds`:
+altExp(filtered_sce)$discard
+altExp(processed_sce)$discard
+```
+
+To filter cells based on this column, use the following command, for example on `filtered_sce`:
+```r
+# Filter cells based on ADT-level QC statistics
+filtered_sce <- filtered_sce[, -which(altExp(filtered_sce)$discard)]
+```
+
+In the `processed.rds` object specifically, this information is also recorded as values `"Remove"` and `"Keep"`:
+
+```r
+# See which cells should be removed vs. kept in `processed.rds`
+processed_sce$adt_scpca_filter
+```
+
+To filter cells based on this column, use the following command:
+```r
+# Filter cells based on ADT-level QC statistics
+processed_sce <- processed_sce[, which(processed_sce$adt_scpca_filter ==  "Keep")]
+```
+
+
+### Normalizing ADT counts
+
+The `filtered.rds` object contains a raw ADT counts matrix, but not a matrix of log normalized ADT expression.
+To perform normalization yourself, we first recommend filtering cells flagged for removal by ADT-level QC statistics to reduce the chances of normalization failing, as described above.
+
+
+Normalization can then be performed with `scuttle` and `scater`:
+
+```r
+# Calculate size factors for use in ADT normalization,
+# using the previously-calculated ambient profile
+altExp(filtered_sce) <- scuttle::computeMedianFactors(
+  altExp(filtered_sce),
+  reference = metadata(altExp(filtered_sce))$ambient_profile
+)
+
+# Normalize and log transform
+# This step will only succeed if all size factors calculated above are positive
+altExp(filtered_sce) <- scater::logNormCounts(altExp(filtered_sce))
+```
+
+A normalized expression matrix is also provided in the `processed.rds` object, containing values for all cells which are indicated as `"Keep"` in the `processed_sce$adt_scpca_filter` column and `NA` for all cells indicated as `"Remove"`:
+
+```r
+# Access log normalized ADT expression matrix
+logcounts(altExp(processed_sce))
+```
```

Comment:
I would also switch this. Talk about the `processed.rds` file first and then you can say if instead you are working with the `filtered.rds` file this is what you would do. We want people to use the `processed.rds` file. Also this should be a follow up from the previous step so you filtered and now you have the normalized data. What would you do with that normalized data? We talk about PCA/UMAP in the RNA section, so perhaps you could mention or show commands for how you would calculate PCA/UMAP for just the ADT information.

Reply:
I would move everything related to working with the filtered file to a separate sub-section and have the main section be the flow you would follow if you had just downloaded the processed file.